### DISCUSSION TITLE
Manticore integration

### DISCUSSION THREAD

**Discussion by inferno-umar** [vectordotdev/vector#21630] [General]
Is vector compatible with `manticore` as a sink ?

https://github.com/manticoresoftware/manticoresearch

**Comment by jszwedko** [comment]
It looks like Manticore aims to be Elasticsearch compatible, so I would expect so. It looks like they actually have a blog post about using Vector: https://manticoresearch.com/blog/integration-of-manticore-with-vectordev/
